### DISCUSSION TITLE
4주차 회고

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#116] [회고]
### TRY(PROBLEM의 해결책으로 다음 회고까지 실행할 점)

아래 양식을 복사해 댓글로 작성해주시면 됩니다.

```
### KEEP

### PROBLEM

### TRY
```

**Comment by minyoung0503** [comment]
### KEEP
서로 칭찬과 격려를 아낌없이 한다.
소통이 빠르다.
사소한 말에도 경청해준다.
어려운 문제가 있을 때 적극적으로 도와준다.

### PROBLEM
(나만) 알람을 못 들어서 회의에 5분씩 지각한다 .. 

### TRY
알람 1분 단위로 맞추기

**Comment by devwqc** [comment]
### KEEP
- 팀원의 고민을 해결해 주기 위해서 노력한다.
- 팀 행사가 있을 때 적극적으로 참여한다.
- PR이 잘 올라오고 전체적으로 컨벤션을 잘 맞추고 있다.

### PROBLEM
- 금주까지 페이지 UI를 완료하기로 했는데 나는 아직 공통 컴포넌트 작업을 진행하고 있다.
- 실수로 EC2 인스턴스를 삭제했다.

### TRY
- 우선순위를 나누어 프로젝트 일정 맞추기.
- EC2 인스턴스 삭제 방지 켜두기.

**Comment by Seoin02** [comment]
### KEEP
- 팀원의 문제를 함께 해결한다.
- 팀원들이 열심히 작업한다.
- 공통 컴포넌트 사용예시를 잘 작성해서 사용하기 편하다.
- FE, BE, Disigner의 진행 상황이 잘 공유되고 있다.

### PROBLEM
- FE 페이지가 많아서 목표한 기간보다 페이지 작업이 오래 걸릴 듯하다.

### TRY
- 그래도 작업 일정을 넉넉히 잡아둬서 다행이다. 다들 화이팅 합시다!
